internal/quickfort/orders.lua

```diff
@@ -166,3 +166,18 @@ function enqueue_orders(stats, buildings, building_db, ctx)
         end
     end
 end
+
+function enqueue_container_orders(ctx, num_bins, num_barrels, num_wheelbarrows)
+    local order_specs = ctx.order_specs or {}
+    ctx.order_specs = order_specs
+    if num_bins and num_bins > 0 then
+        inc_order_spec(order_specs, num_bins, reactions, "wooden bin")
+    end
+    if num_barrels and num_barrels > 0 then
+        inc_order_spec(order_specs, num_barrels, reactions, "rock pot")
```

Comment:
I'm wondering if these might make sense to make configurable, but I think you've chosen reasonable defaults here.

Reply:
I made similar material choices for the regular building blueprint orders. In the docs, I just wrote that if the material doesn't make sense for your fort, go to the manager orders screen and replace the order with a similar one that makes sense. I can definitely make it more configurable if we get complaints, though.